Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS build
+
+WORKDIR /app
+
+COPY . /app
+
+FROM python:${PYTHON_VERSION} AS run
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=build /app .
+
+RUN pip install -r requirements.txt
```

Comment:
Ensure that the requirements.txt file exists in the copied directory. If not, the pip install command will fail.